src/commonMain/resources/db/migration/V71__SearchPlaylistRankedFilters.sql

```diff
@@ -0,0 +1,10 @@
+UPDATE public.playlist
+SET config =
+        jsonb_set(
+                config::jsonb,
+                '{"searchParams", "ranked"}'::text[],
+                (case when config -> 'searchParams' ->> 'ranked' = 'true' then '"Ranked"' else '"All"' end)::jsonb,
```

Comment:
The test playlist inserted in `V62.1__TestSearchPlaylist.sql` has the value explicitly set to `null`, which can't be read as an enum. This accounts for that, but probably isn't needed for the live site.

I didn't know setting it to false could filter *out* ranked maps. I can add that as an option in the enum if you'd like, it would be easy enough.